website/src/components/EventInfo.vue

```diff
@@ -36,7 +36,9 @@
               </span>
               <span class="info-text">
                 <a target="_blank" :href="event.location.url">{{event.location.title}}</a>
-                <span class="info-extended text-gray f6" v-if="event.location.detail">{{event.location.detail}}</span>
+                <span class="info-extended text-gray f6" v-if="event.location.detail">
+                  <markdown :text="event.location.detail"></markdown>
```

Comment:
To avoid rendering paragraphs in an inline context, `markdown-it` supports rendering in “inline mode.” Instead of calling `md.render(text)`, you can call `md.renderInline(text)`, then no `<p>` tags will be created.

What we can do here is to (e.g.) pass a prop `inline` to the Markdown component, so that it switches to inline mode. (I think it’s also okay to create a MarkdownInline component :P)